db/migrate/20250524174213_normalize_speaker_website.rb

```diff
@@ -0,0 +1,14 @@
+class NormalizeSpeakerWebsite < ActiveRecord::Migration[8.0]
+  def change
+    # Fix an edge case where twitter and website are in the same field
+    speaker = Speaker.find_by(slug: "derrick-ko")
+    speaker.update(website: "http://derrickko.com", twitter: "http://twitter.com/derrickko")
```

Comment:
I just fixed this in production. Can we remove this and/or update the static YAML file instead?

Reply:
No worries, thank you! 🙏🏼